howso/direct/client.py

```diff
@@ -4663,18 +4434,18 @@ def react_into_trainee(
             Total sample size of model to use (using sampling with replacement)
             for all non-robust computation. Defaults to 1000.
             If specified overrides sample_model_fraction.```
-        residuals : bool, optional
-            For each context_feature, use the full set of all other
-            context_features to predict the feature. When True computes and
-            caches MAE (mean absolute error), R^2, RMSE (root mean squared
-            error), and Spearman Coefficient for continuous features, and
-            MAE, accuracy, precision, recall, and Matthews correlation coefficient
-            for nominal features. False removes cached values.
-        residuals_robust : bool, optional
-            For each context_feature, computes and caches the same stats as
-            residuals but using the robust (power set/permutations) set of all
-            other context_features to predict the feature.
-            False removes cached values.
+        residuals_hyperparameter_feature : string, optional
```

Comment:
I see `string, optional` here and `str, optional` just below. Honestly not sure which is correct, but fix whichever the more incorrect one is please